internal/broker/bind_create_test.go

```diff
@@ -272,6 +275,58 @@ func TestCreateBindingEndpoint(t *testing.T) {
 		}`, fixture.PlanId, customExpirationSeconds), t)
 		require.Equal(t, http.StatusBadRequest, response.StatusCode)
 	})
+
+	t.Run("should return 404 for not existing binding", func(t *testing.T) {
+		// given
+		instanceID := "1"
+		bindingID := uuid.New().String()
+		endpoint := fmt.Sprintf("v2/service_instances/%s/service_bindings/%s?accepts_incomplete=false", instanceID, bindingID)
+
+		// when
+		response := CallAPI(httpServer, "GET", endpoint, "", t)
+
+		// then
+		require.Equal(t, http.StatusNotFound, response.StatusCode)
+	})
+
+	t.Run("should return created kubeconfig", func(t *testing.T) {
+		// given
+		instanceID := "1"
+		bindingID := uuid.New().String()
+		endpoint := fmt.Sprintf("v2/service_instances/%s/service_bindings/%s?accepts_incomplete=false", instanceID, bindingID)
+		body := fmt.Sprintf(`
+		{
+			"service_id": "123",
+			"plan_id": "%s",
+			"parameters": {	
+				"service_account": true	
+				}	
+		}`, fixture.PlanId)
+
+		// when
+		response := CallAPI(httpServer, "PUT", endpoint, body, t)
+		require.Equal(t, http.StatusCreated, response.StatusCode)
+
+		response = CallAPI(httpServer, "GET", endpoint, "", t)
+
+		// then
+		require.Equal(t, http.StatusOK, response.StatusCode)
+
+		binding := unmarshal(t, response)
+
+		credentials, ok := binding.Credentials.(map[string]interface{})
+		require.True(t, ok)
+		kubeconfig := credentials["kubeconfig"].(string)
+
+		duration, err := getTokenDuration(t, kubeconfig)
+		require.NoError(t, err)
+		assert.Equal(t, expirationSeconds*time.Second, duration)
+
+		//// verify connectivity using kubeconfig from the generated binding
```

Comment:
Why four slashes?

Reply:
To make it different from two slashes.

Reply:
Is this intentional?
Could you reveal the meaning hence it different from meaning of two slashes (beginning of the comment).

Reply:
I wanted to make visible as a subsection of `// given`, `// when`, `// then`